<ui version="4.0" >
 <class>OptTheme</class>
 <widget class="QWidget" name="OptTheme" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptThemeUI</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Chat dialogs</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="useHtmlViewInMucCK" >
        <property name="text" >
         <string>Use HTML chat dialogs for MUC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useHtmlViewInChatsCK" >
        <property name="text" >
         <string>Use HTML chat dialogs for conversations</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox18" >
     <property name="title" >
      <string>Theme</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QComboBox" name="themeCB" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox17" >
     <property name="title" >
      <string>Variant</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>10</number>
      </property>
      <property name="topMargin" >
       <number>10</number>
      </property>
      <property name="rightMargin" >
       <number>10</number>
      </property>
      <property name="bottomMargin" >
       <number>10</number>
      </property>
      <item>
       <widget class="QComboBox" name="variantCB" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
